addons/web/static/src/js/chrome/keyboard_navigation_mixin.js

```diff
@@ -44,6 +45,21 @@ odoo.define('web.KeyboardNavigationMixin', function (require) {
             this._areAccessKeyVisible = false;
             this.BrowserDetection = new BrowserDetection();
         },
+        /**
+         * @override
+         */
+        start: function () {
+            this._hideAccessKeyOverlay = () => this._hideAccessKeyOverlay();
```

Comment:
When I suggest code directly through GitHub markdown, it does not mean that you don't have to test it ;) Branch does not work anymore.

Reply:
My bad, infinite recursion. Fixed.